examples/Learning_example/learning_example.ipynb

```diff
@@ -0,0 +1,2295 @@
+{
```

Comment:
<p>As in the other plot, for paper I would suggest to:</p><p> </p><p>* dpi=150,300 (this should also fix the rather small text size right now)</p><p>* add legend on what the vertical lines are</p><p>* hypothetically you could also use orbital revolutions as x axis to have a more interpretable time axis? Would also fit other plots in the paper. Orbital period for the orbits I provided is 5061.492804584637 seconds.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>